setup.cfg

```diff
@@ -23,7 +23,9 @@ classifiers =
 
 [options]
 py_modules = pyupgrade
-install_requires = tokenize-rt>=3.0.1
+install_requires =
+    tokenize-rt>=3.0.1
+    typing; python_version=="2.7"
```

Comment:
can you run `setup-cfg-fmt` on this, it should bump `python_requires` to `>= 3.4` so that we don't need to also install typing for that

Reply:
I opened a new PR for it